app/models/changelog_data.php

```diff
@@ -52,9 +52,173 @@
+            'section' => ['end_user'],
+            'type'    => ['bug'],
+            'issues'  => [655],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Fix missing spaces between words on some views'],
+            'section' => ['end_user'],
+            'type'    => ['bug'],
+            'issues'  => [658],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Menu tooltip text default value is wrong '],
+            'section' => ['end_user'],
+            'type'    => ['bug'],
+            'commit'  => ['19aa45d9c5a5b7b4bb9f8ae71bb204b2cb262f7a'],
+            'authors' => ['tchevalier'],
+        ],
+        [
+            'message' => ['Multi-word searches only searched as exact string, not multiple strings'],
+            'section' => ['end_user'],
+            'type'    => ['regression'],
+            'issues'  => [644],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Update README and CONTRIBUTING instructions for developers'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'issues'  => [608],
+            'authors' => ['SkySymbol'],
+        ],
+        [
+            'message' => ['Use table head instead of a special row for column headers'],
+            'section' => ['code'],
+            'type'    => ['better'],
+            'commit'  => ['7786a92a3af2507e77d7781ebf7680630ee615da'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Add script to list/delete obsolete files'],
+            'section' => ['code'],
+            'type'    => ['better'],
+            'commit'  => ['405250548bf4f363cfe317703f95ad216b3e8a00'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['Fixes in health status view'],
+            'section' => ['code'],
+            'type'    => ['better'],
+            'commit'  => ['a3c97f25f97b74e16d5fa34c99baa2b864130612'],
+            'authors' => ['tchevalier'],
+        ],
+        [
+            'message' => ['Use prefer-dist unstead of prefer-source for Composer in Travis'],
+            'section' => ['code'],
+            'type'    => ['better'],
+            'issues'  => [677],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Improve security of our GitHub Web hook'],
+            'section' => ['code'],
+            'type'    => ['better'],
+            'issues'  => [680],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Use more meaningful name for section description'],
+            'section' => ['code'],
+            'type'    => ['better'],
+            'commit'  => ['dd2b843f0ee4a730e013b3a64c3c858044e04ae2'],
+            'authors' => ['flod'],
+        ],
+        [
+            'message' => ['[unlocalized strings] Sanitize English words before displaying, fix first row style'],
```

Comment:
This is the same as 'Use table head instead of a special row for column headers', I'd keep that one.